src/static_analysis/manifest.rs

```diff
@@ -2935,7 +2935,7 @@ impl<'de> Deserialize<'de> for Permission {
 
         match deser_result {
             toml::value::Value::String(ref str) => {
```

Comment:
Once again, I think the proper thing would be to change the name of the variable. `str` is a standard library type.